lib/kitchen/figure_element.rb

```diff
@@ -20,7 +20,25 @@ def initialize(node:, document: nil)
     # @return [Element, nil]
     #
     def caption
-      first('figcaption')
+      first('figcaption', "div[data-type='description']")
+    end
+
+    # Returns true if the figure is a child of another figure
+    #
+    # @return [Boolean]
+    #
+
```

Comment:
I think generally no space here; the last blank line with a `#` can be right up against the method, ditto below